File: API/Models/CertificadoSessaoModel.cs
namespace API_Visitatus.Models
{
    public class CertificadoSessaoModel
    {
        public CertificadoDadosLojaModel? objCertificadoDadosLojaModel { get; set; }
        public List<CertificadoDadosPresencaModel>? objCertificadoDadosPresencaModel { get; set; }
        public List<GestaoAdmAtivaModel>? lstCargosGestaoLoja { get; set; }
    }
}

File: Sistema/src/app/models/CertificadoSessao.Model.ts
import { CertificadoDadosLojaModel } from './CertificadoDadosLoja.Model';
import { CertificadoDadosPresencaModel } from './CertificadoDadosPresenca.Model';
import { GestaoAdmAtivaModel } from './GestaoAdmAtiva.Model';

export class CertificadoSessaoModel {
  objCertificadoDadosLojaModel: CertificadoDadosLojaModel | null;
  objCertificadoDadosPresencaModel: CertificadoDadosPresencaModel[] | null;
  lstCargosGestaoLoja: GestaoAdmAtivaModel[] | null;

  constructor() {
    this.objCertificadoDadosLojaModel = null;
    this.objCertificadoDadosPresencaModel = null;
    this.lstCargosGestaoLoja = null;
  }
}
